lisa/notifiers/junit.py

```diff
@@ -28,6 +28,12 @@ class JUnitSchema(schema.Notifier):
     path: str = "lisa.junit.xml"
     # respect the original behavior, include subtest by default
     include_subtest: bool = True
+    # show passed case 'message' and 'stacktraces'
+    include_passed_messages: bool = False
+    # show 'message' for failed cases and 'stacktrace'
+    include_failed_messages: bool = True
```

Comment:
When should it be set to False? If no scenario, it can be included always.

Reply:
Default behavior is to include messages and stacktraces for Failed tests/subcases.
Ideally we want messages for skipped tests.
Passed tests/cases should be optional and extension can bundle additional information in stacktrace

Reply:
I don't see the scenarios, which want set failed/skippped to false, and set passed to true. So the three flags are not needed. The logic always include stackstrace for failed and skipped, but never for passed.

Reply:
understood. I will modify code to only implement logic for successful test cases.